### PR TITLE
Add input labels to Login & Signup pages

### PR DESCRIPTION
**Pull request by patrick-du**
## Description

Link to ticket: https://www.notion.so/uwblueprintexecs/Add-labels-to-inputs-log-in-page-f02c4e3418464a2f9d986c8ba134bf0c

I removed the placeholders because I thought it looked odd but I can put it back if we want both label and placeholder.

### CODE REVIEW

**Review by jzxhuang — COMMENTED** [review]
hmm, placeholders look ok to me in the [Figma mock ](https://www.figma.com/file/s6xLJftM99Xto7FW9AAtGU/Building-Up---Dashboard?node-id=2349%3A753).

I'm pretty indifferent, you can ask designers if you want I guess. otherwise code looks good

**Comment by patrick-du** [comment]
> hmm, placeholders look ok to me in the [Figma mock ](https://www.figma.com/file/s6xLJftM99Xto7FW9AAtGU/Building-Up---Dashboard?node-id=2349%3A753).
> 
> I'm pretty indifferent, you can ask designers if you want I guess. otherwise code looks good

Oh, the placeholders were just "Email" and "Password" before - the Figma does looks better.

File: src/components/dashboard/LoginRegister/RegisterInput.js [hunk 1]
```diff
@@ -50,19 +50,45 @@ const RegisterInput = () => {
...
         <FormControl id="password">
           <FormLabel>Password</FormLabel>
-          <Input type="password" name="password" value={password} onChange={onChangePass} isRequired />
+          <Input
+            type="password"
+            placeholder="8+ characters"
+            name="password"
+            value={password}
+            onChange={onChangePass}
+            isRequired
```
Comment (jzxhuang) [inline src/components/dashboard/LoginRegister/RegisterInput.js:90]:
```suggestion
            isRequired
            minLength={8}
```

apparently this attribute exists, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/text, may as well add it while we're at it

File: src/components/dashboard/LoginRegister/RegisterInput.js [hunk 2]
```diff
@@ -50,19 +50,45 @@ const RegisterInput = () => {
         </Heading>
         <FormControl id="firstName">
           <FormLabel>First Name</FormLabel>
-          <Input name="firstName" value={firstName} onChange={onChangeFirstName} isRequired />
+          <Input
+            name="firstName"
+            placeholder="Enter first name"
+            value={firstName}
+            placeholderonChange={onChangeFirstName}
```
Comment (jzxhuang) [inline src/components/dashboard/LoginRegister/RegisterInput.js]:
```suggestion
            onChange={onChangeFirstName}
```

**Review by jzxhuang — CHANGES_REQUESTED** [review]

**Review by jzxhuang — APPROVED** [review]

**Pull request merged**